test/polyline2.cc

```diff
@@ -52,72 +61,101 @@ TEST(Polyline2, TestGeneralizeSimplification) {
   EXPECT_EQ(line, (Polyline2<Point2>{{{17, 0}, {17, 4}, {17, 5}}}))
       << "Should have removed all but the first, last and marked points";
 
-  Polyline2<PointLL> line1{
```

Comment:
its not easy to tell what is different in these tests. is it formatting or did we need to change some of the numbers to get it to work or are there new tests? maybe all 3? just a mini call out as to what is going on here for the lazy among us :D